src/main/java/gov/cms/mat/cql_elm_translation/service/EffectiveDataRequirementService.java

```diff
@@ -36,45 +29,6 @@ private RequestData createDefaultRequestData(String cql) {
         .build();
   }
 
-  /**
-   * @param libraryDetails - instance of CqlLibraryDetails
-   * @param recursive- indicates if the data requirements gathered should be recursive
-   * @param accessToken- used by MadieLibrarySourceProvider to make calls to cql-library-services
-   * @return effective data requirement of type R5 library
-   */
-  public org.hl7.fhir.r5.model.Library getEffectiveDataRequirements(
```

Comment:
we can delete this service and its unit test completely. this is specific to FHIR/QICore